code/defines/vox_sounds.dm

```diff
@@ -523,7 +523,7 @@ var/list/vox_sounds = list("," = 'sound/vox_fem/,.ogg',
 "personnel" = 'sound/vox_fem/personnel.ogg',
 "pipe" = 'sound/vox_fem/pipe.ogg',
 "plant" = 'sound/vox_fem/plant.ogg',
-"plasma" = 'sound/vox_fem/plasma.ogg',
+"phoron" = 'sound/vox_fem/plasma.ogg',
```

Comment:
Er, these vox sounds are the ones used by AI announcements, right? wouldn't that mean and AI that announces "phoron" would say "plasma"